isis/appdata/import/PDS3/MroHirise.tpl

```diff
@@ -0,0 +1,99 @@
+    SecMirrorMtrRngTemperature      = {{TEMPERATURE_PARAMETERS.MRO_SEC_MIRROR_MTR_RNG_TEMPERATURE.Value}}
+    SecMirrorTemperature            = {{TEMPERATURE_PARAMETERS.MRO_SEC_MIRROR_TEMPERATURE.Value}}
+    SecMirrorBaffleTemperature      = {{TEMPERATURE_PARAMETERS.MRO_SEC_MIRROR_BAFFLE_TEMPERATURE.Value}}
+    IeaTemperature                  = {{TEMPERATURE_PARAMETERS.MRO_IEA_TEMPERATURE.Value}}
+    FocusMotorTemperature           = {{TEMPERATURE_PARAMETERS.MRO_FOCUS_MOTOR_TEMPERATURE.Value}}
+    IePwsBoardTemperature           = {{TEMPERATURE_PARAMETERS.MRO_IE_PWS_BOARD_TEMPERATURE.Value}}
+    CpmmPwsBoardTemperature         = {{TEMPERATURE_PARAMETERS.MRO_CPMM_PWS_BOARD_TEMPERATURE.Value}}
+    MechTlmBoardTemperature         = {{TEMPERATURE_PARAMETERS.MRO_MECH_TLM_BOARD_TEMPERATURE.Value}}
+    InstContBoardTemperature        = {{TEMPERATURE_PARAMETERS.MRO_INST_CONT_BOARD_TEMPERATURE.Value}}
+    DllLockedFlag                   = ({{ join(INSTRUMENT_SETTING_PARAMETERS.MRO_DLL_LOCKED_FLAG.Value, ", ")}})
+    DllResetCount                   = {{INSTRUMENT_SETTING_PARAMETERS.MRO_DLL_RESET_COUNT.Value}}
+    DllLockedOnceFlag               = ({{ join(INSTRUMENT_SETTING_PARAMETERS.MRO_DLL_LOCKED_ONCE_FLAG.Value, ", ")}})
+    DllFrequenceCorrectCount        = {{INSTRUMENT_SETTING_PARAMETERS.MRO_DLL_FREQUENCY_CORRECT_COUNT.Value}}
+    ADCTimingSetting                = ({{ join(INSTRUMENT_SETTING_PARAMETERS.MRO_ADC_TIMING_SETTINGS.Value, ", ")}})
+  {% endblock %}
+
+{% block additional_groups %}
+  Group = Archive
+    DataSetId                       = {{DATA_SET_ID.Value}}
+    ProduccerId                     = {{PRODUCER_ID.Value}}
+    ObservationId                   = {{OBSERVATION_ID.Value}}
+    ProductId                       = {{PRODUCER_ID.Value}}
+    ProductVersionId                = {{PRODUCT_VERSION_ID.Value}}
+    EdrProductCreationTime          = {{TIME_PARAMETERS.PRODUCT_CREATION_TIME.Value}}
+    RationaleDescription            = "{{RATIONALE_DESC.Value}}"
+    OrbitNumber                     = {{ORBIT_NUMBER.Value}}
+    SoftwareName                    = "{{SOFTWARE_NAME.Value}}"
+    ObservationStartTime            = {{TIME_PARAMETERS.MRO_OBSERVATION_START_TIME.Value}}
+    ReadoutStartTime                = {{TIME_PARAMETERS.MRO_READOUT_START_TIME.Value}}
+    TrimLines                       = {{INSTRUMENT_SETTING_PARAMETERS.MRO_TRIM_LINES.Value}}
+    FelicsCompressionFlag           = {{INSTRUMENT_SETTING_PARAMETERS.MRO_FELICS_COMPRESSION_FLAG.Value}}
+    IdFlightSoftwareName            = {{FLIGHT_SOFTWARE_VERSION_ID.Value}}
+  End_Group
+
+  Group = BandBin
+    FilterName                      = {{INSTRUMENT_SETTING_PARAMETERS.FILTER_NAME.Value}}
+    FilterCenter                    = {{INSTRUMENT_SETTING_PARAMETERS.CENTER_FILTER_WAVELENGTH.Value}}
+    FilterWidth                     = {{INSTRUMENT_SETTING_PARAMETERS.BANDWIDTH.Value}}
```

Comment:
Unfortunately, the bandbin group wasn't area wasn't standardized when this app was written. HiRISE uses Name, Center, Width (i.e., without "Filter") part.